src/betterproto/__init__.py

```diff
@@ -373,6 +380,51 @@ def _serialize_single(
     return bytes(output)
 
 
+def _parse_float(value: Any) -> float:
+    """Parse the given value to a float
+
+    Parameters
+    ----------
+    value : Any
```

Comment:
I wouldn't document this. I also think value shouldn't be Any and rather str

Reply:
I think it can be a float too; will check and update.